odoo/http.py

```diff
@@ -2025,6 +2029,8 @@ def _transactioning(self, func, readonly):
                     ):
                         raise  # bubble up to werkzeug.debug.DebuggedApplication
                     if not hasattr(exc, 'error_response'):
+                        if isinstance(exc, AccessDenied):
+                            exc.suppress_traceback()
```

Comment:
Errors are logged in `http.Application`, we should not remove the stack trace, if we do we won't have any useful information left. Maybe we could refactor this so that the handle error is called only from the application.

Reply:
can't do, was explained IRL